packages/venia-ui/lib/components/AccountMenu/accountMenuItems.js

```diff
@@ -0,0 +1,50 @@
+import React, { Fragment } from 'react';
+import { shape, string } from 'prop-types';
+
+import { Link } from '@magento/venia-drivers';
+import { mergeClasses } from '@magento/venia-ui/lib/classify';
+
+import defaultClasses from './accountMenuItems.css';
+
+const MENU_ITEMS = [
+    { name: 'Order History', url: '' },
+    { name: 'Store Credit & Gift Cards', url: '' },
+    { name: 'Favorites Lists', url: '' },
+    { name: 'Address Book', url: '' },
+    { name: 'Saved Payments', url: '' },
+    { name: 'Communications', url: '' },
+    { name: 'Account Information', url: '' }
+];
+
+const AccountMenuItems = props => {
+    const { handleSignOut } = props;
+
+    const classes = mergeClasses(defaultClasses, props.classes);
+
+    const menuItems = MENU_ITEMS.map(item => {
+        return (
+            <Link className={classes.link} to={item.url} key={item.name}>
+                {item.name}
+            </Link>
+        );
+    });
+
+    return (
+        <Fragment>
+            {menuItems}
+            <button
+                className={classes.signOut}
```

Comment:
Do we need to add `type={'button'}` for buttons that are not `submit` by default? Not a huge deal since there is no form surrounding it, but probably for aesthetics. I am fine either way.

Reply:
Resolved by https://github.com/magento/pwa-studio/commit/77ade11229918fa36064436481a8266fbc48cb92.